classes/question_ui_renderer.php

```diff
@@ -355,27 +367,64 @@ private function resolve_placeholders(): void {
             $cleanoption = $parts[1] ?? 'clean';
```

Comment:
Sorry, fällt mir jetzt erst auf, aber du kannst `strtolower` auch direkt hier anwenden (und dafür später nicht mehr):
```suggestion
            $cleanoption = strtolower($parts[1]) ?? 'clean';
```

Reply:
Ist mir auch aufgefallen, aber ich dachte mir, für `debugging("Unrecognized placeholder cleaning option: '$cleanoption', using 'plain'");` ist es sinnvoller, das Original auszugeben :shrug: